contrib/executor/jmeterd/README.md

```diff
@@ -129,8 +131,50 @@ Testkube should now use the new image for the Executor and you can schedule a Te
 
 1. **MASTER_OVERRIDE_JVM_ARGS / SLAVES_OVERRIDE_JVM_ARGS**: Used to override default memory options for JMeter master/slaves. Example: `MASTER_OVERRIDE_JVM_ARGS=-Xmn256m -Xms512m -Xmx512m`.
 
-2. **SLAVES_COUNT**: Specifies the number of slaves required for distributed JMeter tests. Example: `SLAVES_COUNT=3`.
+2. **SLAVES_COUNT**: Specifies the number of slave pods required for distributed JMeter tests. Example: `SLAVES_COUNT=3`. Default value of `SLAVE_COUNT` is 1.
 
 3. **MASTER_ADDITIONAL_JVM_ARGS / SLAVES_ADDITIONAL_JMETER_ARGS**: Allows exporting additional JVM arguments for slaves/master. Example: `MASTER_ADDITIONAL_JVM_ARGS=-Xmx1024m -Xms512m -XX:MaxMetaspaceSize=256m`.
 
 4. **SLAVES_ADDITIONAL_JMETER_ARGS**: Provides additional JVM arguments for JMeter server / slaves. Example: `SLAVES_ADDITIONAL_JMETER_ARGS=jmeter-server -Jserver.rmi.ssl.disable=true -Dserver_port=60000`.
+
+
+**Read README file of Jmeter to learn about how to run Jmeter test.**
+
+Below guide will provide you the details about how to run a Jmeter test in distribution environment.
+
+1. File option:
+
+   When you provide a test (.jmx) file to `Jmeterd ( Jmeter in distributed environment )`, the executor of `Jmeterd` will spawn number of slaves pods specified by user through `SLAVE_COUNT` Environment Variable as desribed above and run the test on all the slave pods.
+
+2. Git Option:
+   Using Git flow of the executor we can have following benifits of `Jmeterd` which is not possible:
+
+   - User can use additional file required by a particular test like a csv or json file.
```

Comment:
I think `csv` and `json` are abbreviations and should be in capital letters.

Reply:
Yup, Done